### PR TITLE
Done with api integration of issue page

### PR DESCRIPTION
**Pull request by utkarsh4321**
Hii @sachin-duhan26 thanks for this great opportunity. I did API integration according to your requirement as I understand. So Please review this PR. If you found any problem with this please tell me so I will try to fix asap.

### CODE REVIEW

File: issues.html [hunk 1]
```diff
@@ -1,46 +1,59 @@
...
-    <script src="./js/main.js"></script>
+    <script
+      src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+      integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+      crossorigin="anonymous"
+    ></script>
+    <script type="module" src="./js/main.js"></script>
     <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
     <script>
-        AOS.init();
```
Comment (sachin-duhan26) [inline issues.html:42]: replace AOS config with the code snippet given below ⬇️ 
```js
    <script>
        AOS.init({
            duration: 700,
            offset: 180
        });
    </script>
```

File: js/controller/getReposIssues.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import { get } from "../constant.js";
+
+const repos = {
```
Comment (sachin-duhan26) [inline js/controller/getReposIssues.js:3]: we are looking forward to displaying all issues that are there for [learn-opensource](http://learn-opensource.github.io/).
- Do not 🚫  limit the issue only to learn-opensource.github.io repo only!

File: js/main.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import { get_organization_contributors } from "./controller/contributors.js";
 import { get_organization_repos } from "./controller/repos.js";
+import { getRepoIssues } from "./controller/getReposIssues.js";
 
-get_organization_repos();
-get_organization_contributors();
\ No newline at end of file
+if (window.location.pathname.slice(1) === "issues.html") {
+  getRepoIssues();
+} else {
```
Comment (sachin-duhan26) [inline js/main.js:7]: let's use an if statement for the "/" route, we may add more routes in the future.

**Review by sachin-duhan26 — CHANGES_REQUESTED** [review]
Hello @utkarsh4321 
You did a **Terrific Job** ❤️  when it comes to writing readable code 💯 . I believe we need some small changes and a more generalized solution for the issue.html page.

Kindly find the detailed review of the #16
Thanks and cheers 🍻

File: js/controller/getReposIssues.js (hunk 2 above)
Comment (utkarsh4321) [reply js/controller/getReposIssues.js:3]: Hii @sachin-duhan26 on fetching issues pull request related to the issue is also coming so I will avoid it or keep it.

Comment (sachin-duhan26) [reply js/controller/getReposIssues.js:3]: I have resolved the merge conflict for you, Cheers 💯

**Comment by utkarsh4321** [comment]
Hii @sachin-duhan26 I fix the requested changes can you please review my PR. and if there is any problem tell me so I will try to fix asap.

**Review by sachin-duhan26 — APPROVED** [review]

**Pull request merged**